scripts/create_service_account.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+# This script will connect to the vCD using username and password,
+# and go through the steps to create/authorize/activate a service account.
+
```

Comment:
Maybe we could add an early check for `jq`?
Something like
```
! which jq > /dev/null && echo 'Error: No "jq" command found in PATH, please install it' && exit 1
```

Reply:
Nice suggestion, fixed